### PR TITLE
added the first setup

### PR DESCRIPTION
**Pull request by Mov305**
Added all the requirements

1. React router
2. Styles 
3. Forms 
4. Booklist

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+        <a
+          className="inline-flex items-center px-8 py-3 text-[#08ACB4] bg-white border border-white rounded-full hover:bg-transparent hover:text-white active:bg-white/90 focus:outline-none focus:ring transition"
+          href="/"
+        >
+          <span className="text-sm font-medium"> Let's Get Started </span>
+          <svg
+            className="w-5 h-5 ml-3"
+            xmlns="http://www.w3.org/2000/svg"
+            fill="none"
+            viewBox="0 0 24 24"
```
Comment (Ugboaja-Uchechi) [inline src/components/Footer.js:17]: - [ ] Great job with your project and design 👏 . On the `categories` page you should have just a `button` that says `Check Status` as stated in the project requirement 🙏 . You can always go back to work on this project as you like once you're done submitting it for Microverse 😊

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @Mov305, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Footer.js (hunk 1 above)
Comment (peterrobert) [reply src/components/Footer.js:17]: This change has been done well.

**Review by peterrobert — APPROVED** [review]
Hi @Mov305, 

### HIGHLIGHTS

1. Great work with your design. You have done a good work 👏🏿 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**